src/vs/workbench/contrib/externalTerminal/browser/externalTerminal.contribution.ts

```diff
@@ -71,7 +72,7 @@ CommandsRegistry.registerCommand({
 				});
 			} else {
 				distinct(targets.map(({ stat }) => stat!.isDirectory ? stat!.resource.fsPath : dirname(stat!.resource.fsPath))).forEach(cwd => {
-					terminalService!.openTerminal(cwd);
+					terminalService!.openTerminal(config.terminal.external, cwd);
```

Comment:
don't you need to check `config?.terminal?.external` here ?

Reply:
No because it's in the renderer where the schema is registered correctly